event.go

```diff
@@ -360,11 +362,10 @@ func (e *Event) AnErr(key string, err error) *Event {
 	case LogObjectMarshaler:
 		return e.Object(key, m)
 	case error:
-		if m == nil || isNilValue(m) {
+		if isNilValue(m) {
```

Comment:
`case nil:` would have caught an actual `nil`

Reply:
You can have a nil type matching the error interface. See my other message.